libvast/vast/data.hpp

```diff
@@ -15,6 +15,7 @@
 #include "vast/data/integer.hpp"
 #include "vast/defaults.hpp"
 #include "vast/detail/operators.hpp"
+#include "vast/detail/type_list.hpp"
```

Comment:
```suggestion
```

Is this really needed?

Reply:
No. I meant to include `vast/detail/type_traits.hpp` -- this is just a typo. I just pushed a fix so we include `vast/detail/type_traits.hpp` for `detail::is_any_v` definition.